Documentation/cmdref/cilium_bpf_egress.md

```diff
@@ -21,8 +21,5 @@ Manage the egress routing rules
 ### SEE ALSO
 
 * [cilium bpf](../cilium_bpf)	 - Direct access to local BPF maps
-* [cilium bpf egress delete](../cilium_bpf_egress_delete)	 - Delete egress entries
-* [cilium bpf egress get](../cilium_bpf_egress_get)	 - Get egress entries
 * [cilium bpf egress list](../cilium_bpf_egress_list)	 - List egress policy entries
-* [cilium bpf egress update](../cilium_bpf_egress_update)	 - Update egress entries
 
```

Comment:
> because the policy is getting too complex for a CLI-based configuration?

not really, in the end every policy can be distilled down to a list of `(source IP, dest CIDR, gateway IP, egress IP)` entries in the egress policy bpf map (and those commands were used exactly for this, to interact just with the bpf map).

The problem is that after the initial implementation we changed the way the manager reacts to events. Now it:
* keeps an internal state of all the objects it is interested in (nodes, endpoints, egress policies)
* whenever there's a new event (e.g. a new endpoint is added or a policy is updated) it updates its internal state to reflect that
* finally, it [reconciles](https://github.com/cilium/cilium/blob/4b1cb31e0c32e956ebd673a847a02cba63700779/pkg/egressgateway/manager.go#L371-L391) the state of the system with its internal state (so it distills the list of nodes, ep and policies down to a list of entries for the bpf map, adds the ones that are missing and removes the ones that should not be present anymore)

Because of this we should not allow a user to manually interact with the map entries, since whenever there's a new event (for example a new endpoint added) the manager will run its reconciliation logic, and will just delete (or add back) the entries that the user manually created (or deleted)